src/Tickets/Seating/Frontend.php

```diff
@@ -116,31 +116,33 @@ public function print_tickets_block( $html, $file, $name, $template, $context ):
 		}
 
 		$prices   = [];
-		$provider = Tickets::get_event_ticket_provider_object( $post_id );
-		foreach ( tribe_tickets()->where( 'event', $post_id )->get_ids( true ) as $ticket_id ) {
-			$ticket = $provider->get_ticket( $post_id, $ticket_id );
-			if ( ! $ticket ) {
-				continue;
-			}
-			$prices[] = $ticket->price;
-		}
+		$provider = Tickets::get_event_ticket_provider_object( $post_id ); // @todo - i see the below method can return false but we are never checking against it.
+
+		$tickets = array_filter(
+			array_map(
+				static function ( $ticket_id ) use ( $provider, $post_id ) {
+					$ticket = $provider->get_ticket( $post_id, $ticket_id );
+					if ( ! $ticket ) {
+						return false;
+					}
+
+					return $ticket;
+				},
+				tribe_tickets()->where( 'event', $post_id )->get_ids()
+			)
+		);
 
-		if ( ! count( $prices ) ) {
+		if ( empty( $tickets ) ) {
 			// Why are we here at all?
 			return $html;
 		}
 
+		$prices     = wp_list_pluck( $tickets, 'price' );
```

Comment:
Restoring the `foreach` loop this will not be required. It's just more efficient to do that in the loop.